src/test/java/eu/xenit/gradle/docker/compose/DockerComposeConventionImplTest.java

```diff
@@ -228,5 +230,66 @@ public void testFromProjectWithDockerAlfrescoPlugin() {
         assertEquals(DOCKER_IMAGE_ID, composeSettings.getEnvironment().get("SUB_PROJECT1_DOCKER_IMAGE"));
     }
 
+    @Test
+    public void testFromProjectWithEnvironmentVariable(){
+        composeConvention.fromProject("testname",project);
+        TaskCollection<DockerBuildImage> buildImageTaskCollection = project.getTasks().withType(DockerBuildImage.class);
+        assertTaskDependsOn(composeSettings.getUpTask(), dockerBuildImageProvider);
+        assertTaskDependsOn(composeSettings.getBuildTask(), dockerBuildImageProvider);
+        assertTaskDependsOn(composeSettings.getPushTask(), dockerBuildImageProvider);
+
+        runSetupImageEnvironmentWithComposeUp();
+
+        assertEquals(DOCKER_IMAGE_ID, composeSettings.getEnvironment().get("TESTNAME_BUILD_DOCKER_IMAGE"));
```

Comment:
Check that also the `TESTNAME` environment variable exists and has this image ID.

(You will need to apply the `DockerPlugin` for that to work and configure the image ID on that task: https://github.com/xenit-eu/alfresco-docker-gradle-plugin/blob/61038ee7b4c84c38ce774e19ec393c6bea54f0c2/src/test/java/eu/xenit/gradle/docker/compose/DockerComposeConventionImplTest.java#L203-L207)

Reply:
Done in a new test, since it complained that a buildDockerImage already existed and therefore it could not apply DockerPlugin.